examples/json.rs

```diff
@@ -22,7 +24,8 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .header("content-type", "application/json")
         .body(serde_json::to_vec(&request)?)?
         .send()?
-        .json::<ShoutCloudResponse>()?;
+        .json::<ShoutCloudResponse>()?; /* The json method is only available if 
+        you enabled the json feature in your Cargo.toml */
```

Comment:
The comment at the top of the file should be good enough; spamming the code with comments seems a little much.